release-manager/files/README.md

```diff
@@ -60,17 +60,31 @@ repositories:
 
 This type designates ODS components designed for _code development_. Such repositories are based on quickstarters whose names start with `be-`, `ds-`, or `fe-`, for _backend_, _data science_, and _frontend_, respectively. This is the default type.
 
+If you use this type ODS expects to find JUnit XML test results. If you do not have any test results the pipeline will fail. If you are deploying something where JUnit XML test results are not available consider using <<Repository Type: ods-service>>.
+
+#### Repository Type: ods-infra
+
+This type designates ODS components designed for _consuming on-prem or cloud services_ of arbitrary type using infrastructure as code. Such components are based on quickstarters whose names start with `infra-`.
+
+#### Repository Type: ods-saas-service
+
+This type designates ODS components designed for _documenting vendor-provided SaaS services_.
+
 #### Repository Type: ods-service
 
-This type designates ODS components designed for _running some service_. Examples include repositories based on the `be-gateway-nginx` quickstarter.
+This type designates ODS components designed for _running services_ of arbitrary type. Examples include repositories based on the `airflow-cluster` quickstarter.
 
 #### Repository Type: ods-test
 
-This type designates ODS components designed for _running automated tests against a live application_. Such repositories are based on quickstarters whose names start with `e2e-`.
+This type designates ODS components designed for _running automated tests against a live application_. Such components are based on quickstarters whose names start with `e2e-`.
+
+#### Repository Type: ods-library (EXPERIMENTAL feature)
+
+This type designates ODS components designed for _library components_. Such components are only build in dev (WIP and assemble mode), and are needed for other components to be built.
```

Comment:
typo ... are only built vs. build.

It might make sense to add something like:

The component is not built in nor promoted to qa, neither to prod.

Reply:
Yeah, some rephrasing might be good.